mask/eolearn/mask/cloud_mask.py

```diff
@@ -910,3 +624,9 @@ def execute(self, eopatch):
             eopatch.data[multi_proba_feature] = (multi_proba * is_data).astype(np.float32)
 
         return eopatch
+
+
+class AddMultiCloudMaskTask(CloudMaskTask):
```

Comment:
Pylint complains that a class docstring is missing -> [CI report](https://travis-ci.org/github/sentinel-hub/eo-learn/jobs/763471991#L1569)

Reply:
Didn't think about that one as it is deprecated. I've added something just to make pylint happy.